packages/wrangler/src/r2/helpers.ts

```diff
@@ -368,6 +376,20 @@ export function eventNotificationHeaders(
 	}
 	return headers;
 }
+
+export async function getEventNotificationConfig(
+	apiCredentials: ApiCredentials,
+	accountId: string,
+	bucketName: string
+): Promise<GetNotificationConfigResponse> {
+	const headers = eventNotificationHeaders(apiCredentials);
+	logger.log(`Fetching notification configuration for bucket ${bucketName}...`);
+	return await fetchResult<GetNotificationConfigResponse>(
```

Comment:
I am not sure that simply returning the value here will result in the response being logged to the console, if that it is what you expected.

Reply:
Yeah, I have expanded on this so the response is formatted into a table and printed via `logger.table`